vendor/assets/javascripts/jquery.textile.js

```diff
@@ -46,15 +46,27 @@
     this._defaults = defaults;
     this._name = pluginName;
 
-    this.init();
+    this.init(this.$element.data('textiled'));
   }
 
   Plugin.prototype = {
-    init: function () {
+    init: function ( exists ) {
```

Comment:
can we get the `exists` (check `data-textilized`) value here in the constructor and skip passing it?